web3.js/src/stake-program.js

```diff
@@ -484,7 +484,7 @@ export class StakeProgram {
    * Max space of a Stake account
    */
   static get space(): number {
-    return 4008;
+    return 200;
```

Comment:
Please add a comment that justifies how this number was obtained

Reply:
Added